js/Track/Model/File/BIGWIG.js

```diff
@@ -0,0 +1,19 @@
+Genoverse.Track.Model.File.BIGWIG = Genoverse.Track.Model.Graph.Bar.extend({
+
+  getData : function(chr, start, end){
+    var model = this;
+    var deferred = $.Deferred();
+
+    if(!this.url) this.data = new dallianceLib.BlobFetchable(this.track.dataFile);
+    else this.data = new dallianceLib.URLFetchable(this.url);
+
+    new BWReader(this.data, this.name, function(bw){
```

Comment:
* Replace this.data with this.bigwigFile
* We need to cache the BWReader (like the VCFReader is cached) to avoid making requests for the header of the file every time getData is called